### DISCUSSION TITLE
Does Papermerge encrypt my documents?

### DISCUSSION THREAD

**Discussion by tiuub** [ciur/papermerge#366] [Q&A] [answered]
Hello,

I am currently thinking about to build up a self-hosted DMS. One of the most important points, which I want to cover in my system, is encryption. In this repo I couldn't find anything about encryption. So Im asking for help here.

Does Papermerge encrypt my documents and files after the upload or does it keeps the documents unencrypted?

Kind regards
tiuub

**Comment by ciur** [comment, marked as answer]
No, Papermerge does not encrypt documents.
